docs/README.md

```diff
@@ -0,0 +1,3 @@
+# Http Methods Documentation
```

Comment:
If we change the title, it'd need to be changed here as well.

Reply:
Done.